maplibre_gl_web/lib/src/maplibre_web_gl_platform.dart

```diff
@@ -1156,4 +1156,16 @@ class MapLibreMapController extends MapLibrePlatform
   Future<List> getSourceIds() async {
     throw UnimplementedError();
   }
+
+  /// Method to set style string
+  /// [styleString] -> It will take json, url, absolute path or asset path
+  ///
+  @override
+  Future<void> setStyle(String styleString) async {
+    try {
+      _map.setStyle(styleString);
+    } on PlatformException catch (e) {
+      return Future.error(e);
+    }
```

Comment:
When does _map.setStyle throw a PlatformException? (It's not used in the other methods.)